tests/containers/containers.test.js

```diff
@@ -39,9 +41,11 @@ describe('Service Lifecyle Integration Test', () => {
     execSync(`${serverlessExec} create --template-path ${templateName} --path ${tmpDir}`);
     process.chdir(tmpDir);
     execSync('npm i');
```

Comment:
Is `npm i` really useful before `npm link`? (there are multiple occurrences of this in the PR)

Reply:
I think it's not necessary, we can keep `npm link` without `npm i`